inscriptions/ISic000517.xml

```diff
@@ -199,14 +202,29 @@
+                    <lb n="4" break="no"/>blico <g ref="#interpunct">·</g> exornato <g ref="#interpunct">·</g> pa
+                    <lb n="5" break="no"/>trono <g ref="#interpunct">·</g> ob <g ref="#interpunct">·</g> merita <g ref="#interpunct">·</g> eius
+                    <lb n="6"/>quae <g ref="#interpunct">·</g> in <g ref="#interpunct">·</g> cives <g ref="#interpunct">·</g> suos
+                    <lb n="7"/>confert <g ref="#interpunct">·</g> <expan><abbr>c</abbr><ex>larissimo</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>v</abbr><ex>iro</ex></expan> <g ref="#interpunct">·</g> adlecto
+                    <lb n="8"/>inter <g ref="#interpunct">·</g> tribunicios
+                    <lb n="9"/>ab <g ref="#interpunct">·</g> <expan><abbr>Imp</abbr><ex>eratore</ex></expan> <g ref="#interpunct">·</g> Caesare
+                    <lb n="10"/><del rend="erasure"><name><expan><abbr>M</abbr><ex>arco</ex></expan></name> <g ref="#interpunct">·</g> <name><expan><abbr>Aur</abbr><ex>elio</ex></expan></name> <g ref="#interpunct">·</g> <name>Commodo</name> <g ref="#interpunct">·</g></del> <name>Anto
+                    <lb n="11" break="no"/>nino</name> <g ref="#interpunct">·</g> Pio <g ref="#interpunct">·</g> Felice
+                    <lb n="12"/><expan><abbr>Aug</abbr><ex>usto</ex></expan> <g ref="#interpunct">·</g> tribules <g ref="#interpunct">·</g> <expan><abbr>t<hi rend="ligature">ri</hi>b</abbr><ex>us</ex></expan>
+                    <lb n="13"/>Iovis <g ref="#interpunct">·</g> <expan><abbr>Aug</abbr><ex>usti</ex></expan> <g ref="#interpunct">·</g> pecu<hi rend="ligature">ni</hi>a
+                    <lb n="14"/>sua <g ref="#interpunct">·</g> <expan><abbr>l</abbr><ex>ocus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>p</abbr><ex>ublice</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>d</abbr><ex>atus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>d</abbr><ex>ecreto</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>d</abbr><ex>ecurionum</ex></expan>
+                </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp><app><note>Text: CIL</note></app></listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#AA">
+                <p>To Marcus Marcius Bieti Glaucus, son of Marcus, of tribe Claudia, father of a senator, honoured with the public horse and patron on account of his merit, which the most renowned man devoted to the citizens, adlected amongst the tribunes by the Imperator Caesar Marcus Aurelius Commodus Antoninus Pius Felix Augustus, the tribesmen of tribe Jupiter Augustus [gave this] with their own money. This place was given by decree of the decurions.</p>
```

Comment:
Bietis is the nominative (weird name, I know); 'on account of his merits which he conferred upon his fellow citizens, vir clarissimus, etc.' 'This place was given at public expense by...'